### PR TITLE
v3.8.0-4 round: fix unlink prefab bug in inspector

### PR DESCRIPTION
**Pull request by arsen2010**
Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: editor/inspector/contributions/node.js [hunk 1]
```diff
@@ -637,10 +636,20 @@ const Elements = {
...
+                for (const dump of panel.dumps) {
+                    const prefab = dump.__prefab__;
+                    switch (role) {
+                        case 'unlink':
+                        case 'reset': {
+                            recordings.push(prefab.rootUuid);
+                        }
+                    }
+                }
+                const undoID = await beginRecording(recordings);
```
Comment (dogeFu) [inline editor/inspector/contributions/node.js]: unlink-prefab不需要在业务层记录undo，场景中unlink时已经记录好了https://github.com/cocos/editor-extensions/pull/595

File: editor/inspector/contributions/node.js [hunk 2]
```diff
@@ -637,10 +636,19 @@ const Elements = {
...
+                const recordings = [];
+                for (const dump of panel.dumps) {
+                    const prefab = dump.__prefab__;
+                    switch (role) {
+                        case 'reset': {
+                            recordings.push(prefab.rootUuid);
+                        }
+                    }
+                }
+                const undoID = await beginRecording(recordings);
```
Comment (dogeFu) [inline editor/inspector/contributions/node.js]: 这里如果recordings没有uuid最好不要beginRecording

Comment (arsen2010) [reply editor/inspector/contributions/node.js]: done modified

**Review by dogeFu — APPROVED** [review]

**Comment by dogeFu** [comment]
https://github.com/cocos/3d-tasks/issues/16518

**Review by VisualSJ — APPROVED** [review]

**Pull request merged**